client/src/util/dataframe/dataframe.js

```diff
@@ -273,7 +290,14 @@ class Dataframe {
     const columns = [...this.__columns];
     columns.push(colData);
     const colIndex = this.colIndex.withLabel(label);
-    return new this.constructor(dims, columns, rowIndex, colIndex);
+    const columnsAccessor = [...this.__columnsAccessor, undefined];
```

Comment:
[low priority] Maybe add a comment here? This was a little inscrutable to me at first, trying to figure out why we were adding undefined. Actually you don't need to push to the end since you are already using the fact that js returns undefined if you try an out of bounds index on an array.

Reply:
good point - I originally added it to be clear about the expected dimensions, but it doesn't add anything of value.   Will remove and rely in implicit array access behavior.